card/lib/card/spec_helper.rb

```diff
@@ -76,4 +73,43 @@ def render_card_with_args view, card_args={}, format_args={}, view_args={}
   def users
     SharedData::USERS.sort
   end
+
+  # Make expectations in the event phase.
+  # Takes the usual event options :on and :before/:after/:around
+  # and registers the event_block with these options as an event.
+  # Unknown methods in the event_block are executed in the rspec context
+  # instead of the card's context.
+  # An additionaly :trigger block in opts is expected that is called
+  # to start the event phase.
+  # Example:
+  # in_phase before: :approve, on: :save,
+  #          trigger: ->{ test_card.update_attributes! content: '' } do
+  #            expect(item_names).to eq []
+  #          end
+  def in_phase opts, &event_block
```

Comment:
Assignment Branch Condition size for in_phase is too high. [19.75/15]<br>Method has too many lines. [24/15]